transmogrifier/sources/ead.py

```diff
@@ -452,3 +424,59 @@ def parse_mixed_value(
         elif isinstance(item, Tag) and item.name not in skipped_elements:
             for child in item.children:
                 yield from cls.parse_mixed_value(child, skipped_elements)
+
+    def parse_dates(
+        self, collection_description_did: Tag, source_record_id: str
+    ) -> list[timdex.Date]:
+        """
+        Dedicated method to parse dates.  Targeting archdesc.unitdata elements, using
+        only those with a @normal attribute value.  These are almost uniformly ranges,
+        but in the event they are not (or two identical values for the range) a single
+        date value is produced.
+        """
+
+        dates = []
+        for date_element in collection_description_did.find_all("unitdate"):
+            normal_date = date_element.get("normal", "").strip()
+            if normal_date == "":
+                continue
+
+            date_instance = timdex.Date()
+
+            # date range
+            if "/" in normal_date:
+                gte_date, lte_date = normal_date.split("/")
+                if gte_date != lte_date:
+                    if validate_date_range(
+                        gte_date,
+                        lte_date,
+                        source_record_id,
+                    ):
+                        date_instance.range = timdex.Date_Range(
+                            gte=gte_date,
+                            lte=lte_date,
+                        )
+                else:
+                    date_str = gte_date  # arbitrarily take one
```

Comment:
Optional, probably doesn't need to be assigned to another var with the `# arbitarily take one` note, you could just use `gte` with that note

Reply:
I'd considered that, but given it's used twice below, figured one line / one comment would inform the lines below.  Unless feel strongly, inclined to leave as-is.